docs/mongocli/command/mongocli-atlas-privateEndpoints-azure-interfaces-describe.txt

```diff
@@ -19,10 +19,26 @@ Syntax
 
 .. code-block::
 
-   mongocli atlas privateEndpoints azure interfaces describe <endpointId> [options]
+   mongocli atlas privateEndpoints azure interfaces describe <endpointServiceId> [options]
 
 .. Code end marker, please don't delete this comment
 
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - endpointServiceId
```

Comment:
Do we need to remove this as an option on line 53 of this file if we make endpointServiceId the argument? Same question for all of the privateEndpoint interface commands.

Reply:
good question, I'm not sure, this set of command on private endpoint are weird and the way they were coded doesn't match how they were documented, I need to test them out to see how they really work